mclagsyncd/mclaglink.cpp

```diff
@@ -0,0 +1,814 @@
+            type = "dynamic";
+        else
+            type = "static";
+
+        /*get port name*/
+        mclagsyncd_get_oid_2_port_name_map(oid_2_portname_map);
+#if 0
+        SWSS_LOG_NOTICE("oid_2_portname_map:");
+        {
+            std::unordered_map<std::string, std:: string>::iterator it;
+
+            for (it = oid_2_portname_map.begin(); it != oid_2_portname_map.end(); ++it)
+                SWSS_LOG_NOTICE("%s vs %s", it->first.c_str(), it->second.c_str());
+        }
+#endif
+        mclagsyncd_get_bridgePortId_2_attrPortId_map(&brPortId_2_attrPortId_map);
+#if 0
+        SWSS_LOG_NOTICE("brPortId_2_attrPortId_map:");
+        {
+            std::map<std::string, std:: string>::iterator it;
+
+            for (it = brPortId_2_attrPortId_map.begin(); it != brPortId_2_attrPortId_map.end(); ++it)
+                SWSS_LOG_NOTICE("%s vs %s", it->first.c_str(), it->second.c_str());
+        }
+#endif
+        brPortId_it = hash.find("SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID");
+        if (brPortId_it == hash.end())
+        {
+            continue;
+        }
+        bri_port_id = brPortId_it->second;
+
+        brPortId_2_attrPortId_it = brPortId_2_attrPortId_map.find(bri_port_id);
+        if (brPortId_2_attrPortId_it == brPortId_2_attrPortId_map.end())
+        {
+            continue;
+        }
+
+        oid_2_portName_it = oid_2_portname_map.find(brPortId_2_attrPortId_it->second);
+        if (oid_2_portName_it == oid_2_portname_map.end())
+        {
+            continue;
+        }
+
+        port_name = oid_2_portName_it->second;
+
+        /*insert set*/
+        SWSS_LOG_NOTICE("read one fdb entry(mac:%s, vid:%d, port_name:%s, type:%s) from ASIC_DB and insert new_set.", mac.c_str(), vid, port_name.c_str(), type.c_str());
+        fdb_set->insert(mclag_fdb(mac, vid, port_name, type));
+    }
+
+    return;
+}
+
+void MclagLink::mclagsyncd_set_port_isolate(char *msg)
+{
+    mclag_sub_option_hdr_t *op_hdr = NULL;
+    string isolate_src_port;
+    string isolate_dst_port;
+    char * cur = NULL;
+    string acl_name = "mclag";
+    string acl_rule_name = "mclag:mclag";
+    vector<FieldValueTuple> acl_attrs;
+    vector<FieldValueTuple> acl_rule_attrs;
+    std::string acl_key = std::string("") + APP_ACL_TABLE_TABLE_NAME + ":" + acl_name;
+    std::string acl_rule_key = std::string("") + APP_ACL_RULE_TABLE_NAME + ":" + acl_rule_name;
+
+    cur = msg;
+
+    /*get isolate src port infor*/
+    op_hdr = (mclag_sub_option_hdr_t *)cur;
+    cur = cur + MCLAG_SUB_OPTION_HDR_LEN;
+    isolate_src_port.insert(0, (const char*)cur, op_hdr->op_len);
+
+    cur = cur + op_hdr->op_len;
+
+    /*get isolate dst ports infor*/
+    op_hdr = (mclag_sub_option_hdr_t *)cur;
+    cur = cur + MCLAG_SUB_OPTION_HDR_LEN;
+    isolate_dst_port.insert(0, (const char*)cur, op_hdr->op_len);
+
+    SWSS_LOG_NOTICE("Enter mclagsyncd_set_port_isolate");
```

Comment:
you shall use `SWSS_LOG_ENTER`

Reply:
Sure , we will remove some unnecessary log.